clusterloader2/run-e2e.sh

```diff
@@ -22,5 +22,18 @@ CLUSTERLOADER_ROOT=$(dirname "${BASH_SOURCE}")
 export KUBECONFIG="${KUBECONFIG:-${HOME}/.kube/config}"
 export KUBEMARK_ROOT_KUBECONFIG="${KUBEMARK_ROOT_KUBECONFIG:-${HOME}/.kube/config}"
 
+# Deploy the GCP PD CSI Driver is required
+if [ $1 = "--deploy-csi-driver" ]; then
+   if [[ -z "${E2E_GOOGLE_APPLICATION_CREDENTIALS}" ]]; then
```

Comment:
:-

otherwise you're risking that this will fail with unbounded variable

Reply:
ack